cmd/fs-v1-metadata.go

```diff
@@ -315,37 +317,63 @@ func createFormatFS(fsPath string) error {
 }
 
 func initFormatFS(fsPath string) (rlk *lock.RLockedFile, err error) {
+	// Done channel is used to close any lingering retry routine, as soon
```

Comment:
What do you mean by separate process?  What is the limitation of adding unit tests?